src/scripts/import-open-api.ts

```diff
@@ -195,12 +195,14 @@ export const getResReqTypes = (
       if (isReference(res)) {
         return getRef(res.$ref);
       } else {
-        if (res.content && res.content["application/json"]) {
-          const schema = res.content["application/json"].schema!;
-          return resolveValue(schema);
-        } else if (res.content && res.content["application/octet-stream"]) {
-          const schema = res.content["application/octet-stream"].schema!;
-          return resolveValue(schema);
+        if (res.content) {
+          for (let contentType of Object.keys(res.content)) {
+            if (contentType.startsWith("application/json") || contentType.startsWith("application/octet-stream")) {
+              const schema = res.content[contentType].schema!;
+              return resolveValue(schema);
+            }
+          }
+          return "void";
         } else {
           return "void";
         }
```

Comment:
Since we are touching this part of the code, let's improve the readability a little bit by decreasing the amount of levels of code indentation. I suggest:

```typescript
export const getResReqTypes = (
  responsesOrRequests: Array<[string, ResponseObject | ReferenceObject | RequestBodyObject]>,
) =>
  uniq(
    responsesOrRequests.map(([_, res]) => {
      if (!res) {
        return "void";
      }

      if (isReference(res)) {
        return getRef(res.$ref);
      }

      if (res.content) {
        for (let contentType of Object.keys(res.content)) {
          if (contentType.startsWith("application/json") || contentType.startsWith("application/octet-stream")) {
            const schema = res.content[contentType].schema!;
            return resolveValue(schema);
          }
        }
        return "void";
      }

      return "void";
    }),
  ).join(" | ");
```